### PR TITLE
Task-48819: Fix the mentioning users on chat application.

### PR DESCRIPTION
**Pull request by ahmadmido**
**Issue:** 1-can't mentioning users on the chat application, the problem is caused by a promise exception to have the updated user avatar.
2-The user's full name isn't set on the user bean object on getRoomParticipantsToSuggest.
**Solution:** Make the user avatar service on the render item method and update the user's full name on the user bean object.

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue [hunk 1]
```diff
@@ -185,10 +185,11 @@ export default {
...
-          const avatar = this.userAvatar;
+          chatServices.getUserInfo(item.name).then((data) => {
+            this.userAvatar = data.avatar ;
+          });
           const defaultAvatar = '/chat/img/room-default.jpg';
-          return `<img src="${avatar}" onerror="this.src='${defaultAvatar}'" width="20px" height="20px">
+          return `<img src="${this.userAvatar}" onerror="this.src='${defaultAvatar}'" width="20px" height="20px">
                       ${chatServices.escapeHtml(item.fullname)}<span style="float: right" class="chat-status-task chat-status-'+item.status+'"></span>`;
         },
         /* eslint-disable no-template-curly-in-string */
```
Comment (AzmiTouil) [inline application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue:195]: Is this change needed ?

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue (hunk 1 above)
Comment (ahmadmido) [reply application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue:195]: > Is this change needed ?

Yes, is needed to have the value of the usersAvatar without having an exception with the promised result. discussed with @AzmiTouil

**Review by AzmiTouil — APPROVED** [review]
LGTM
